### PR TITLE
Footnotes

### PR DESCRIPTION
**Pull request by mongoKart**
Staging: https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/footnotes/reference/footnotes/

Summarized slide 30 of the markup presentation and added a link to a page from docs-ecosystem as an example.

### CODE REVIEW

File: source/reference/footnotes.txt [hunk 1]
```diff
@@ -0,0 +1,25 @@
+=========
+Footnotes
+=========
+
+Limit footnotes to tangential explanations of greater technical depth that build
+understanding but can be ignored.
```
Comment (kyuan-mongodb) [inline source/reference/footnotes.txt:6]:
```suggestion
Footnotes allow you to explain topics in greater detail,  but can be ignored.
```
Suggestion: I think this can be simplified to something similar to the suggestion above-- as is, its a bit wordy and tough to understand.

File: source/reference/footnotes.txt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Use ``[#footnote-identifier]`` to mark the footnote location in the text, then add the
```
Comment (kyuan-mongodb) [inline source/reference/footnotes.txt:8]: Suggestion: Lets convert this to a procedure on how to create a footnote -- here's my draft of [the hyperlink page](https://github.com/mongodb/docs-meta/pull/115) (I still need to make some formatting edits, but I think a procedure would simplify the process of whoever is trying to learn it)

**Review by kyuan-mongodb — CHANGES_REQUESTED** [review]
Great start! Leaving some suggestions to further simplify the page.

File: source/reference/footnotes.txt [hunk 3]
```diff
@@ -0,0 +1,44 @@
+=========
+Footnotes
+=========
+
+Use footnotes to add detail that not all readers will need.
```
Comment (kyuan-mongodb) [inline source/reference/footnotes.txt:5]: Suggestion: I think its worth mentioning that footnotes render at the very bottom of the page later. And in your example below, you can add a few lines of regular text to really showcase how it works.

**Review by kyuan-mongodb — CHANGES_REQUESTED** [review]
Super close -- Just a suggestion to make this page even stronger!

File: source/reference/footnotes.txt (hunk 3 above)
Comment (mongoKart) [reply source/reference/footnotes.txt:5]: Just making sure I understand:
1. I think footnotes render wherever you put the Footnotes rubric, which is why I mentioned putting it at the bottom of the page.
2. You mean adding some sample text in the example before the footnote rubric?

Comment (kyuan-mongodb) [reply source/reference/footnotes.txt:5]: Yes! both correct! I just figured some dummy text between where you create the foodnote reference and the actual footnote section really showcases how to use them.

Comment (mongoKart) [reply source/reference/footnotes.txt:5]: Is lorem ipsum okay? Or is there a preferred way to create dummy text?

Comment (kyuan-mongodb) [reply source/reference/footnotes.txt:5]: Yea that'd be fine. Or to make it even simpler, you can make it something like: 

<your footnote references>

This describes something cool.
This goes into more detail. 
Now you've learned this cool thing!

<the footnotes rendered>

Comment (mongoKart) [reply source/reference/footnotes.txt:5]: love it

File: source/reference/footnotes.txt [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
```
Comment (kyuan-mongodb) [inline source/reference/footnotes.txt:5]:
```suggestion
Use footnotes to add details that not all readers will need.
```

File: source/reference/footnotes.txt [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+    .. step:: Mark the footnote location in your text with ``[#footnote-identifier]``,
+        where "footnote-identifier" is arbitrary. 
+
+    .. step:: If the bottom of the page doesn't have a "Footnotes" rubric, use 
+        ``.. rubric:: Footnotes`` to create one.
+    
+    .. step:: Beneath the "Footnotes" rubric, enter  ``.. [#footnote-identifier] 
+                <footnote text>`` to add your footnote.
+
```
Comment (kyuan-mongodb) [inline source/reference/footnotes.txt:20]: Since this describes the steps in how to create one, lets put a codeblock to show what these steps describe.

**Review by kyuan-mongodb — APPROVED** [review]
LGTM % tiny suggestions! Great Work!
